sites/public/src/components/applications/ApplicationMultiselectQuestionStep.tsx

```diff
@@ -162,15 +208,32 @@ const ApplicationMultiselectQuestionStep = ({
         <FormBackLink
           url={conductor.determinePreviousUrl()}
           onClick={() => {
-            conductor.setNavigatedBack(true)
-            setPage(page - 1)
+            if (!verifyAddress) {
+              conductor.setNavigatedBack(true)
+              setPage(page - 1)
+              body.current = null
+            }
           }}
           custom={page !== 1}
         />
 
         <div className="form-card__lead border-b flex flex-col items-center">
-          <h2 className="form-card__title is-borderless">{strings?.title ?? question?.text}</h2>
-          {strings?.subTitle && <p className="field-note mt-6">{strings?.subTitle}</p>}
+          <h2 className="form-card__title is-borderless">
+            {verifyAddress
+              ? foundAddress.invalid
+                ? t("application.contact.couldntLocateAddress")
+                : t("application.contact.verifyAddressTitle")
+              : strings?.title ?? question?.text}
+          </h2>
+          {verifyAddress && body.current.options.filter((option) => option.checked).length > 1 && (
+            <p className="field-note mt-6">
+              Since there are multiple options for this preference, you’ll need to verify multiple
+              addresses.
```

Comment:
Can you add this to the translation file? We will add the non-english translations as a followup ticket